### ISSUE TITLE
Perivascular inflammation of pre-capillary pulmonary arteries 

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6521]
**Preferred term label:** Perivascular inflammation of pre-capillary pulmonary arteries 

**Synonyms**
Perivascular inflammation of arterioles

**Definition (free text, please give PubMed ID)**
Perivascular infiltration of precapillary pulmonary arteries (arterioles; 20-70 um) with inflammatory cells

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary artery morphology HP:0030966

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777
 

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Altered the definition to 
Inflammatory cell infiltration surrounding precapillary pulmonary arteries (arterioles; 20-70 micrometer). 
please reopen if needed @eswietlik
